src/modules/layout/components/Toolbar.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import React from 'react'
+import type { Node } from 'react'
+import cx from 'classnames'
+import style from './Toolbar.css'
+
+type Props = {
+  className?: string,
+  children?: Node
```

Comment:
Shouldn't this be `Array<Node>`?

Reply:
I think Node also can be an array of nodes. But not sure.